Gopkg.lock

```diff
@@ -1496,7 +1496,7 @@
   version = "v0.1.0"
 
 [[projects]]
-  digest = "1:199c4a3508ac049896ac06f518c99113e8404723ecf1891dd0c81c3cc17ba410"
+  digest = "1:e5e6165f043c38f641355912f5bf94134531e82abceaa5f086a9008f1bea6655"
```

Comment:
i think this change is accidental? maybe reverting this file and rebasing will get rid of this unintended change

Reply:
Yup, unintended change. The file has been reverted.